README.md

```diff
@@ -71,22 +71,21 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+This code would be best placed in a model method. According to semantic coding guidelines, the model should house any code that relates to a particular table and does not involve routing or basic CRUD functionality.
 ```
```

Comment:
This would actually be best defined as a helper method. We use helper methods when we're defining code that influences how things are displayed in the browser (i.e., affect how HTML is rendered).

Model methods influence how information is taken in and out of the database.